src/OpenColorIO/ops/fixedfunction/FixedFunctionOpGPU.cpp

```diff
@@ -192,6 +192,177 @@ void Add_Glow_03_Inv_Shader(GpuShaderText & ss, float glowGain, float glowMid)
     ss.newLine() << "outColor.rgb = outColor.rgb * glowGainOut + outColor.rgb;";
 }
 
+void Add_GamutComp_13_Compress(GpuShaderText & ss,
+                              const char * dist,
+                              const char * comprDist,
+                              float lim,
+                              float thr,
+                              float power,
+                              float invert)
+{
+    float one_over_power = 1.f / power;
+
+    ss.newLine() << "{";
+    ss.indent();
+ 
+    // No compression below threshold.
+    ss.newLine() << "if (" << dist << " < " << thr << ")";
+    ss.newLine() << "{";
+    ss.indent();
+    ss.newLine() << comprDist << " = " << dist << ";";
+    ss.dedent();
+    ss.newLine() << "}"; // if (dist < thr)
+    
+    // Compress.
```

Comment:
Minor: Regarding my other comment about using a period at the end of sentences, I only meant that this is just the general convention of the code, but please don't feel like you have to do that even for single words.  Just use your best judgement about what looks good to you.  :)

Reply:
That was a bit extreme, point noted :)